R/doAzureParallel.R

```diff
@@ -181,14 +195,24 @@ setVerbose <- function(value = FALSE) {
   assign("packages", obj$packages, .doAzureBatchGlobals)
   assign("pkgName", pkgName, .doAzureBatchGlobals)
 
-  time <- format(Sys.time(), "%Y%m%d%H%M%S", tz = "GMT")
-  id <-  sprintf("%s%s",
-                 "job",
-                 time)
-
   if (!is.null(obj$options$azure$job)) {
     id <- obj$options$azure$job
   }
+  else {
+    time <- format(Sys.time(), "%Y%m%d%H%M%S", tz = "GMT")
+    id <-  sprintf("%s%s",
+                   "job",
```

Comment:
Lines 202 and 203 have very different styles. One has all the params on one line, the other has it split (for roughly the same number of characters on a single line). Why?

Reply:
Not sure why, I updated line 203 to 1 line, and did a reformat again and it was kept in one line, the inconsistency might be due to manual update.